.env

```diff
@@ -0,0 +1,4 @@
+REACT_APP_VX_ENABLE_WRITE_IN_ADJUDICATION=TRUE
```

Comment:
This is a file generated with all the default options, I'm not sure the best way to keep this up to date but we should do something there longer term.